docs/reference/supervisor.md

```diff
@@ -812,6 +812,45 @@ p' = R * p + T
 
 Where *p* is a point whose coordinates are given with respect to the local coordinate system of a node, *R* the rotation matrix returned by the `wb_supervisor_node_get_orientation` function, *T* is the position returned by the `wb_supervisor_node_get_position` function and *p'* represents the same point but this time with coordinates expressed in the global (world) coordinate system.
 
+%spoiler "**Python Example**: How to calculate relative positions and orientations?"
+
+If you want to calculate the position and orientation of any node relative to any other node, you can take a look at this example in python.
+It should be easily adaptable to any other language, as it uses simple matrix and vector calculations. 
+
+```python
+from controller import Supervisor
+import numpy as np
+
+robot = Supervisor()
+ur10e = robot.getFromDef('ur10e')
+box = robot.getFromDef('box')
+
+# get the transposed rotation matrix of the robot, so we can calculate poses of everything relative to it
```

Comment:
Make comment lines not too long results in a better layout and improved readability.
```suggestion
# Get the transposed rotation matrix of the robot, so we can calculate poses of
# everything relative to it.
```